src/ggrc/converters/query_helper.py

```
@@ -92,7 +92,7 @@ def _find_child_type(self, expr, child_type):
     """Search for child_type recursively down the query expression."""
     if child_type:
       return child_type
-    left, oper, right = expr.get("left"), expr.get("op"), expr.get("right")
+    left, oper, right = expr.get("left"), expr.get("op", {}), expr.get("right")
```

@pod2metra You are correct, and the fact that later you expect something that is not there in the first place is a good enough reason to just throw the error here, because if OP needs to be a dict, that means that OP also needs to be set. Therefore, not having OP set should throw an error and prevent us from allowing invalid queries.

We have no idea what will happen with some future operators and if some code would allow empty dict as an operator. So to be safe, when we encounter an error (which this is) we should report it and not silently just continue and hide possible bugs elsewhere where developers are trying to use this function.